feedback.py

```diff
@@ -1,56 +1,70 @@
+        }
 
-            feedback_view = FeedbackView(post_score)
-            message_content = f"<@{user_id}>, on a scale of 1 to 5, how helpful was this conversation?"
-            await self._send_message(channel_id=thread_id, message=message_content, view=feedback_view)
+    async def __call__(self, *args):
+        return await self.ta_feedback(*args)
 
-            try:
-                feedback_score = await asyncio.wait_for(feedback_queue.get(), timeout=60 * 30)
-                await self._send_message(thread_id, f'Thank you for your feedback!')
+    async def ta_feedback(self, guild_id, thread_id, user_id):
+        """
+        Takes thread_id, sends it to the ta-channel, collect's feedback
+        """
 
-            except asyncio.TimeoutError:
-                await self._send_message(thread_id, '*Feedback time out.*')
-                feedback_score = 'na'
+        feedback_channel_id = self._feedback_config['channel_id']
+
+        async with queue("feedback", None) as feedback_queue:
 
-            await self._record_feedback(guild_id, thread_id, user_id, feedback_score)
+            message_content = f"<@{self._feedback_config['ta_role_id']}>, on a scale of 1 to 5, how helpful was this conversation https://discord.com/channels/{guild_id}/{thread_id}/{user_id} (Add your reaction below)"
 
+            # check to see if this a discord.message
+            message_id = await self._send_message(feedback_channel_id, message_content)
+            feedback_message = await self._fetch_message(feedback_channel_id, message_id)
 
-class FeedbackButton(discord.ui.Button):
-    def __init__(self, label, post_score):
-        super().__init__(label=label, style=discord.ButtonStyle.grey)
-        self.post_score = post_score
+            # TODO - this is where the workflow alias would be registered
 
-    async def callback(self, interaction: discord.Interaction):
-        for button in self.view.children:
-            button.disabled = True
-        self.view.stop()
-        await interaction.response.edit_message(view=self.view)
+            for reaction in self._reactions:
+                await feedback_message.add_reaction(reaction)
+                await asyncio.sleep(0.5)  # per discord policy, we wait
 
-        # Last thing is to post the feedback, which closes the thread
-        feedback_score = self.label
-        await self.post_score(feedback_score)
+            try:
+                # TODO - Actually grab the emoji we need
+                feedback_emoji = await asyncio.wait_for(feedback_queue.get(), timeout = 60 * 60 * 24 * 7) # fix this to work because feedback_queue doesn't have a get method.
+                feedback_score = self._reactions[feedback_emoji]
+                # TODO - add a checkmark reaction to the message
+                await feedback_message.add_reaction('✅')
 
+            except asyncio.TimeoutError:
+                await self._send_message(feedback_channel_id, '*Feedback time out.*')
```

Comment:
Instead of this message, add an ❌ reaction (so the TAs know that message can no longer be reviewed)